pyproject.toml

```diff
@@ -1,7 +1,105 @@
+[project]
+name = "extension-helpers"
+authors = [{name = "The Astropy Developers", email = "astropy.team@gmail.com"}]
+license = {text = "BSD 3-Clause License"}
+description = "Utilities for building and installing packages with compiled extensions"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Framework :: Setuptools Plugin",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: System :: Archiving :: Packaging",
+]
+requires-python = ">=3.7"
```

Comment:
Do we really need Python 3.8 support going forward?

Reply:
3.7 being EOL I think it's okay to drop it. 3.8 may still be used downstream however, and if it's not an actual burden why not keep it ?

Reply:
Bumping to 3.8